gtsam.h

```diff
@@ -2723,6 +2723,11 @@ virtual class EssentialMatrixFactor : gtsam::NoiseModelFactor {
 };
 
 #include <gtsam/slam/dataset.h>
+class SfM_data {
```

Comment:
Seems rather limited usefulness if you can't actually get any data from it...

Reply:
Yeah I was wondering about that but I just needed something quick to get my A->B of testing for the projections.

Reply:
Yes, but that means it will need to be added before we can merge this PR

Reply:
Yes, but that means it will need to be added before we can merge this PR